doc/tools/yamllint.md

```diff
@@ -8,12 +8,12 @@
 
  Run yamllint in a file
 ```sh
-yamllint path_to_file
+yamllint -c path/to/config-yamllint.yaml/in/server path_to_file
```

Comment:
Why don't you put relative path to the config file you have commited ?

`yamllint -c ../../server/scratchy_server/config-yamllint.yaml path_to_file`

Reply:
cause you could be in any directory when you call yamllint

Reply:
Well at least mention the path with full location `server/scratchy_server/config-yamllint.yaml` might be more understandable the `stuff/in/server` is a bit confusing

Reply:
Done in 
94e05e3